token.go

```diff
@@ -141,19 +143,19 @@ func (r *TokenRefreshResponse) Populate(reader io.Reader) error {
 }
 
 // TokenStore for ecobee Access and Refresh tokens.
-type TokenStore interface {
+type TokenStorer interface {
 	// AccessToken gets the access token from the store.
-	AccessToken() string
+	GetAccessToken() (string, error)
```

Comment:
We don't really _need_ the `Get` prefix, since the functions are pretty clear in function already and it's shorter without it. I'll leave it up to you whether you want to leave it.